R/mobr.R

```diff
@@ -473,6 +503,18 @@ get_plot_dens = function(comm, density_stat){
     return(plot_dens)   
 }
 
+#' Auxillary function for effect_ functions
+#' Compute an overall p-value for one factor in the discrete case
+#' p-value is based on mean squared difference from zero summed across the scales
+#' @keywords internal
```

Comment:
I think the difference from Loosmore & Ford is that my implementation doesn't take the observed deviation as one sample along with the deviation from the null. So in our code it's `/length(dev_null)`, while in Loosmore & Ford I believe it was `/(length(dev_null) - 1)`. Does that make sense? I recall this is sth that we have discussed with Nick.